### ISSUE TITLE
Loading plugin models causing SasView to crash

### ISSUE THREAD

**Issue by DTWToolan** [SasView/sasview#1621]
I've recently updated SasView versions to 4.2.2. & 5.0.2 (Mac OS versions) and now when a plugin model is selected in the fit window, SasView crashes.

I've previously been successfully using plugin models on 4.2.2. earlier in the week and after opening 5.0.2 the problem started to occur (in both 4.2.2. and 5.0.2 ). 

I've tried deleting both SasView versions and the hidden /.SasView/ folder, in order to resolve the problem but this has not worked. 

Many thanks,

Dan

**Comment by smk78** [comment]
Can you upload the plugin, please. Dan.
Steve

**Comment by DTWToolan** [comment]
Hi Steve,

There are two in the attached zip that I've just made for testing if they work using the sum|multiply(p1, p2) builder, one is a sum and the other a multiplication. As I say, all the plugins where working correctly until I opened 5.0.2 earlier in the week.

Other than the hidden /.SasView/ folder where the plugins are saved and the configuration file is, are there any other folders / files that would be hidden elsewhere? I assume that I must have missed something in attempting a clean install. Just to add, everything is working fine on 4.2.2. on another (very old machine).

[plugin_models.zip](https://github.com/SasView/sasview/files/4895386/plugin_models.zip)
